src/lib/model/ng-package-data.ts

```diff
@@ -53,6 +54,9 @@ export class NgPackageData {
       this.libExternals = ngPackageConfig.lib.externals;
       this.flatModuleFileName = ngPackageConfig.lib.flatModuleFile;
       this.entryFile = ngPackageConfig.lib.entryFile;
+      if (ngPackageConfig.lib.jsx) {
```

Comment:
This if check is not necessary.

Reply:
Ahh right, an empty string would still be falsey for the `ngPkg.jsxConfig` later on. I'll update shortly.